glusterd2/config.go

```diff
@@ -44,6 +44,7 @@ func parseFlags() {
 	flag.String("workdir", "", "Working directory for GlusterD. (default: current directory)")
 	flag.String("localstatedir", "", "Directory to store local state information. (default: workdir)")
 	flag.String("rundir", defaultRunDir, "Directory to store runtime data.")
+	flag.String("hooksdir", "", "Directory to store Hook scripts.")
```

Comment:
I'd prefer that this not be made configurable. I don't see a use-case for making this option configurable because there are pre-designated directories (suc as `create/post`) inside hooks dir.